core/revised/ch-inventory.tex

```diff
@@ -119,7 +114,7 @@ \subsubsection{Light Swords / Knives}
 Fast weapons that deal \telem{Puncture}-elemental damage.  Their Offensive and Deffensive Stats are Air.  Due to their weak damage, they are largely a defensive option.  While a character is equipped with one of these weapons, they may use the reaction \taction{Parry}.  It is used when they suffer a physical attack to make a roll Air vs Air at difficulty 40.  If successful, they don't suffer the attack's effects.  These weapons are always Melee.
 
 \begin{tabwpn}[label=inv-lsword,range=melee,type=physical,element=puncture,roll=airvair]
-    \tabwpnrow[name=Femme Fatale,mlevel=64,cost=Artifact,damage=16x,effect=\tstatus{Death} touch \newline \telem{Shadow}-elemental damage]
+    \input{revised/generated/wpn-light-swords}
 \end{tabwpn}
 
```

Comment:
The other weapon types aside from light weapons aren't here. Theres still heavy weapon & shield, heavy weapons, polearms, claw/gloves, katanas, bows, crossbow/rifles, throwing weapons, wands, staves, instruments and twin weapons to be added,  Do you want me to convert them to .tex format?